ffi/derive/tests/ui_fail/derive_skip_field.rs

```diff
@@ -15,16 +15,16 @@ pub struct FfiStruct {
 fn main() {
     let s: *mut _ = &mut FfiStruct { a: 42, b: 32 };
 
-    let a = MaybeUninit::<*const u32>::uninit();
-    let b = MaybeUninit::<*const u32>::uninit();
+    let mut a = MaybeUninit::<*const u32>::uninit();
+    let mut b = MaybeUninit::<*const u32>::uninit();
```

Comment:
Why does it need to be mutable now? Because setters now writes result to them or what?